### ISSUE TITLE
RAM and CPU Usage layers not working

### ISSUE THREAD

**Issue by DUKENNSCHMI** [antonpup/Aurora#1018]
### What are you experiencing an issue with?

- [ ] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [x] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [ ] Crash
- [ ] Performance problems
- [x] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
The Layer should change the color of the keys if the CPU/RAM usage is rising (Percent Effect Layer; CPUUsage; max value 100, progressive (gradual)

### Actual behavior:
The assigned keys stay at the backlight color and don't change even after I created a new profile

### How often does this problem occur?

This problem occurs... since today (maybe an update has rolled)

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. 
2. 
3. 

### What is the version of Aurora you are experiencing this issue with?

My version is... beta v06.0.0b

### Does the previous version of Aurora have this issue?
not sure if there was an update, if yes then no

### Please include any logs, screenshots, or files that are related to this issue:

For logs, you can use: http://pastebin.com/
For screenshots, you can use: http://imgur.com/
For files, you can use: http://www.tinyupload.com/ , https://www.dropbox.com/ , or attach them to the issue.

**Comment by simon-wh** [comment]
That is a strange issue, have you tested on [v0.6.1-dev4](https://github.com/antonpup/Aurora/releases) to see if you experience the same thing? If so, could you post the latest log file from that version?

**Comment by DUKENNSCHMI** [comment]
I tried the dev version like you said but the problem still occurs 

log: https://pastebin.com/tZMDqZ2s

**Comment by simon-wh** [comment]
I've just started experiencing this on my machine and I don't know why it has suddenly started. My log is full of errors, but it is likely the same issue, I'll try and get this fixed tonight

**Comment by DUKENNSCHMI** [comment]
okay, so it's not only me... good to know
but I can't seem to find what is causing the error.
btw. my log is about 10k lines and it's the same error over and over

thanks for your help ^^

**Comment by simon-wh** [comment]
Sorry for the delay, have been busy. Could you verify that the error you've been getting is the same as this?
```
[ERROR] 20:56:17: Exception: System.TypeInitializationException: The type initializer for 'Aurora.Profiles.LocalPCInformation' threw an exception. ---> System.InvalidOperationException: Category does not exist.
   at System.Diagnostics.PerformanceCounterLib.CounterExists(String machine, String category, String counter)
   at System.Diagnostics.PerformanceCounter.InitializeImpl()
   at System.Diagnostics.PerformanceCounter..ctor(String categoryName, String counterName, String instanceName, Boolean readOnly)
   at System.Diagnostics.PerformanceCounter..ctor(String categoryName, String counterName, String instanceName)
   at Aurora.Profiles.LocalPCInformation..cctor() in D:\Repos\Aurora\Project-Aurora\Project-Aurora\Profiles\GameState.cs:line 205
   --- End of inner exception stack trace ---
   at Aurora.Profiles.LocalPCInformation..ctor()
   at Aurora.Profiles.GameState`1.get_LocalPCInfo() in D:\Repos\Aurora\Project-Aurora\Project-Aurora\Profiles\GameState.cs:line 58
   at lambda_method(Closure , GameState )
   at Aurora.Profiles.StringProperty`1.GetValueFromString(String name, Object input) in D:\Repos\Aurora\Project-Aurora\Project-Aurora\Profiles\StringProperty.cs:line 98
   at Aurora.Utils.GameStateUtils.RetrieveGameStateParameter(IGameState state, String parameter_path, Object[] input_values) in D:\Repos\Aurora\Project-Aurora\Project-Aurora\Utils\GameStateUtils.cs:line 138
```
I only had this come up on one occassion and hasn't happened since, so it might not be the exact same thing causing your issue.

**Comment by DUKENNSCHMI** [comment]
No Problem, but yes its mainly the same error message:

_[ERROR] 21:59:47: Exception: System.TypeInitializationException: Der Typeninitialisierer für "Aurora.Profiles.LocalPCInformation" hat eine Ausnahme verursacht. ---> System.InvalidOperationException: Die Kategorie ist nicht vorhanden.
   bei System.Diagnostics.PerformanceCounterLib.CounterExists(String machine, String category, String counter)
   bei System.Diagnostics.PerformanceCounter.InitializeImpl()
   bei System.Diagnostics.PerformanceCounter..ctor(String categoryName, String counterName, String instanceName, Boolean readOnly)
   bei System.Diagnostics.PerformanceCounter..ctor(String categoryName, String counterName, String instanceName)
   bei Aurora.Profiles.LocalPCInformation..cctor() in C:\projects\aurora\Project-Aurora\Project-Aurora\Profiles\GameState.cs:Zeile 205.
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei Aurora.Profiles.LocalPCInformation..ctor() in C:\projects\aurora\Project-Aurora\Project-Aurora\Profiles\GameState.cs:Zeile 232.
   bei Aurora.Profiles.GameState`1.get_LocalPCInfo() in C:\projects\aurora\Project-Aurora\Project-Aurora\Profiles\GameState.cs:Zeile 58.
   bei lambda_method(Closure , GameState )
   bei Aurora.Utils.GameStateUtils.RetrieveGameStateParameter(IGameState state, String parameter_path, Object[] input_values) in C:\projects\aurora\Project-Aurora\Project-Aurora\Utils\GameStateUtils.cs:Zeile 140._

just in the last line, it differentiates

**Comment by simon-wh** [comment]
@DUKENNSCHMI Could you look at the solution [here](https://stackoverflow.com/a/34615451) for that error to see if that solves the problem for you?

**Comment by DUKENNSCHMI** [comment]
Seems like it fixed it after I restarted my computer, but I can't say how long it will work ...

Thanks for the workaround ^^
Greetings

**Comment by simon-wh** [comment]
@DUKENNSCHMI Were you running your computer straight that whole time? If so, that's probably the issue, when I had it, it only appeared once and disappeared after a restart.

**Comment by DUKENNSCHMI** [comment]
You mean after the fix or before?
Before the fix, I restarted many many times, only after the fix, a restart worked

Greetings

**Comment by simon-wh** [comment]
@DUKENNSCHMI Ah, okay. Glad you got it fixed.
